src/postprocessors/TotalNeutronLeakage.C

```diff
@@ -0,0 +1,52 @@
+#include "TotalNeutronLeakage.h"
+
+registerMooseObject("MoltresApp", TotalNeutronLeakage);
+
+InputParameters
+TotalNeutronLeakage::validParams()
+{
+  InputParameters params = SideIntegralPostprocessor::validParams();
+  params += ScalarTransportBase::validParams();
+  params.addClassDescription("Postprocessor for computing total neutron leakage along provided "
+                             "boundaries for all neutron group fluxes");
+  params.addRequiredParam<unsigned int>("num_groups", "The total number of energy groups");
+  params.addRequiredCoupledVar("group_fluxes", "All the variables that hold the group fluxes. "
+                                               "These MUST be listed by decreasing "
+                                               "energy/increasing group number.");
+  return params;
+}
+
+TotalNeutronLeakage::TotalNeutronLeakage(const InputParameters & parameters)
+  : SideIntegralPostprocessor(parameters),
+    ScalarTransportBase(parameters),
+    _vars(getCoupledMooseVars()),
+    _diffcoef(getMaterialProperty<std::vector<Real>>("diffcoef")),
+    _num_groups(getParam<unsigned int>("num_groups"))
+{
+  addMooseVariableDependency(_vars);
+  unsigned int n = coupledComponents("group_fluxes");
+  if (!(n == _num_groups))
```

Comment:
How come you didn't use:
```suggestion
  if (n != _num_groups)
```

Reply:
Yea you're right. Good catch!